### ISSUE TITLE
Error message: An error occured fetching the depositor's inventory.

### ISSUE THREAD

**Issue by gunwin100** [ztizzlegaming/CSGOWinBig-SteamBot#38]
I'm having some problems with running this bot.  I get this error message when I try to deposit skins.
"Error message:
An error occured fetching the depositor's inventory."

My account is well over 15 days old now.  I should no longer have any trade restrictions.
Is it possible that my account got trade banned because I manually refused a few trades?
How would I know if there was a trade ban on my account?

Any idea what could be causing this?

**Comment by lowheartrate** [comment]
I don't think this has anything to do with the actual Steam account settings. Have you made sure you changed the default code to your bots inventory [here](https://github.com/ztizzlegaming/CSGOWinBig-SteamBot/blob/master/SteamBot/DepositTradeOfferUserHandler.cs#L231) and [here](https://github.com/ztizzlegaming/CSGOWinBig-SteamBot/blob/master/SteamBot/DepositTradeOfferUserHandler.cs/#L408)?

**Comment by gunwin100** [comment]
Yes I made sure to change the default code as you mentioned.  I replaced the number in those urls with my bots own steam number.  
I still get this error message.

**Comment by wwcam** [comment]
It's a response message from check-items.php, so I'm not sure what's causing it.
